frontend/src/Schedulr.js

```diff
@@ -15,38 +15,55 @@ class Schedulr extends React.Component {
   constructor(props) {
     super(props);
     this.state = {
-      activeStep: 0,
-      steps: ['Input Courses and Preferences', 'View Schedules', 'Export to Google Calendar'],
+      activeStep: (new URLSearchParams(window.location.search).get('signedIn') === 'true') ? 1 : 0,
+      steps: ['Sign In', 'Input Courses and Preferences', 'View Schedules', 'Export to Google Calendar'],
       scheduleList: [],
       schedulesTimes: [],
+      calId: '',
+      termDates: {
+        startDate: '',
+        endDate: '',
+      },
     };
 
     this.handleNext = this.handleNext.bind(this);
     this.handleBack = this.handleBack.bind(this);
     this.setScheduleList = this.setScheduleList.bind(this);
     this.generateScheduleMeetingTimes = this.generateScheduleMeetingTimes.bind(this);
+    this.exportToGoogleCalendar = this.exportToGoogleCalendar.bind(this);
+    this.setTermDates = this.setTermDates.bind(this);
   }
 
   getStepContent() {
     const step = this.state.activeStep;
     switch (step) {
     case 0:
+      return (
+        <Button onClick={this.signIn}>Get Started</Button>
```

Comment:
Would it make more sense for this button to say something to the effect of "sign in" with some additional text giving the context around this (ie. we need you to give us google calendar access in order to export the calendars).

Reply:
Yes, I should definitely give more context - I was going to add more of that in later, but I will do that here. Thanks!